index 2b73026..97a9fdb 100644
--- a/src/lanarts_defines.cpp
+++ b/src/lanarts_defines.cpp
@@ -8,13 +8,6 @@
 
 void lanarts_system_quit() {
 	lanarts_net_quit();
-	SDL_QuitSubSystem(SDL_INIT_EVERYTHING ^ SDL_INIT_AUDIO);
-
 	lsound::stop_music();
-#ifndef __LINUX__
-	// Problem with WaitDone for AudioDevice callback using PulseAudio: http://pulseaudio.org/ticket/866
-	// We simply do not quit the audio subsystem on linux ... pity
-	lsound::deinit();
-	SDL_QuitSubSystem(SDL_INIT_AUDIO);
-#endif
+	SDL_Quit();
 }
